src/convert.rs

```diff
@@ -0,0 +1,192 @@
+use num::Float;
+
+use {Alpha, Rgb, Luma, Xyz, Yxy, Lab, Lch, Hsv, Hsl, Color};
+
+///Provides conversion between the colors.
+///Color Conver requires from and into xyz and derives others from this.
```

Comment:
> `///Color Conver`

This looks like a mistake. I would also suggest swapping this line with the line below, to keep that as one paragraph, while removing this sentence from the trait list. I don't know if you have checked how things looks after running `cargo doc`...